mysql-test/mysql-test-run.pl

```diff
@@ -6279,17 +6287,32 @@ sub lldb_arguments {
     return;
   }
 
-  mtr_tofile($lldb_start_file, "lldb -s $lldb_init_file $$exe\n");
-  chmod 0755, $lldb_start_file;
+  my $term_cmd = $ENV{MTR_TERM} || "xterm -title %title% -e";
+  my $lldb_cmd = $ENV{MTR_LLDB} || "lldb";
+  my @term_args = split / /, $term_cmd;
+  my $term_exe = shift @term_args;
 
   $$args= [];
-  mtr_add_arg($$args, "-n");
-  mtr_add_arg($$args, "-W");
-  mtr_add_arg($$args, "-a");
-  mtr_add_arg($$args, "Terminal");
-  mtr_add_arg($$args, $lldb_start_file);
+  foreach my $t_arg (@term_args) {
```

Comment:
THis duplication can be avoided?

Reply:
Fixed, I extracted it to a helper function.